queue/src/fiskaltrust.Middleware.Storage.SQLite/SQLiteStorageBootstrapper.cs

```diff
@@ -64,12 +66,15 @@ private async Task InitAsync(Guid queueId, Dictionary<string, object> configurat
             await PersistMasterDataAsync(baseStorageConfig, _configurationRepository,
                 new SQLiteAccountMasterDataRepository(_connectionFactory, _sqliteFile), new SQLiteOutletMasterDataRepository(_connectionFactory, _sqliteFile),
                 new SQLiteAgencyMasterDataRepository(_connectionFactory, _sqliteFile), new SQLitePosSystemMasterDataRepository(_connectionFactory, _sqliteFile)).ConfigureAwait(false);
-            
+
             var journalFRCopyPayloadRepository = new SQLiteJournalFRCopyPayloadRepository(_connectionFactory, _sqliteFile);
             var journalFRRepository = new SQLiteJournalFRRepository(_connectionFactory, _sqliteFile);
 
-            await PersistMigrationDataAsync(journalFRCopyPayloadRepository, journalFRRepository).ConfigureAwait(false);
-            
+            if (appliedMigrations.Contains(Migrations.JournalFRCopyPayload))
+            {
+                await PerformMigrationInitialization(appliedMigrations, journalFRCopyPayloadRepository, journalFRRepository).ConfigureAwait(false);
+            }
```

Comment:
I think we can just do it like that. If appliedMigrations is empty nothing will happen automatically. And If we add more later on then it will also work with out changes.
```suggestion
            await PerformMigrationInitialization(appliedMigrations, journalFRCopyPayloadRepository, journalFRRepository).ConfigureAwait(false);
```